src/main/java/de/rwth/idsg/steve/repository/dto/WebUserOverview.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * SteVe - SteckdosenVerwaltung - https://github.com/steve-community/steve
+ * Copyright (C) 2013-2024 SteVe Community Team
+ * All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package de.rwth.idsg.steve.repository.dto;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public final class WebUserOverview {
+
+    private final Integer webUserPk;
+    private final Boolean enabled;
+    private final String webusername;
+    private final String[] autorithies;
```

Comment:
typo in field name. and why `String[]`, which can be considered a low-level construct by modern practices, instead of `List<String>`

Reply:
... because of <private String[] fromJson(JSON jsonArray) >

Reply:
you could always convert this array into a list yourself though. here is a helper method that you can put into WebUserService:

```
    private List<String> fromJsonToList(JSON jsonArray) {
        return Arrays.stream(fromJson(jsonArray)).toList();
    }
```

i did the method with String[] because this was the simplest way to do for my use case.

Reply:
I'm a bit old fashion but for array-to-list conversion, I recommend `Arrays.asList()` instead.

Reply:
my refactor commit eliminated the root cause and tackled this from a different perspective. therefore, the need for array-to-list is gone.